.babelrc

```diff
@@ -4,7 +4,7 @@
       "@babel/preset-env",
       {
         "targets": {
-          "node": 6
+          "node": 8
```

Comment:
Latest lts is node 12
In any case this is a breaking change

Reply:
Oh... Then I guess it's probably better that we bump the version in a separate PR as a breaking change. Reverted the target to node v6 for now. https://github.com/Open-Attestation/document-store/pull/99/commits/99b8faccbe2fa5f2ccbdc2670ed789721b993f0a